pytest.ini

```diff
@@ -5,6 +5,9 @@ addopts = --tb=short --numprocesses auto --dist loadgroup
 ; You may need to use the addopts setting below to unbreak debugging in your IDE
 ; addopts = --tb=short
 
+markers =
+    focus: what is being developed right now
```

Comment:
Forgot I left this in here. This allows you to use the `focus` marker that you can apply to individual tests / test classes and then run on their own, rather than having to run the entire test suite.

E.g.

```python
@pytest.mark.focus
def test_something():
  // something
```

And then you can run any tests with that marker using `poetry run pytest -m focus`

Was useful in a previous project where we were running 1700 odd tests which would take ~30 seconds or so. Given the test suite is pretty quick now, I don't think this is as useful though, so I'm happy to remove it